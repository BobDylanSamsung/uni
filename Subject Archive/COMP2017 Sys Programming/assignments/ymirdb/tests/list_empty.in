list keys
list entries
list snapshots
list key
list asdfasdfasdf
bye
