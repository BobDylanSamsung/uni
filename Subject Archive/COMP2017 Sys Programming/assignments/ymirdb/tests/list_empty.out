no keys
no entries
no snapshots
invalid command
invalid command
bye
